jwst/extract_1d/extract.py

```diff
@@ -1439,11 +1439,26 @@ def locn_from_wcs(
         fwd_transform = self.wcs(x, y)
         middle_wl = np.nanmean(fwd_transform[2])
 
-        try:
-            x_y = self.wcs.backward_transform(targ_ra, targ_dec, middle_wl)
-        except NotImplementedError:
-            log.warning("Inverse wcs is not implemented, so can't use target coordinates to get location of spectrum.")
-            return
+        if input_model.meta.exposure.type in ['NRS_FIXEDSLIT', 'NRS_MSASPEC',
+                                              'NRS_BRIGHTOBJ']:
+            if slit is None:
+                xpos = input_model.source_xpos
+                ypos = input_model.source_ypos
+            else:
+                xpos = slit.source_xpos
+                ypos = slit.source_ypos
```

Comment:
Someone mentioned in the JP ticket discussing MOS mode that the "estimated_source_in_slit_x/y" meta values were no longer available in the resampled (s2d) images used as input to extract_1d. I assume these source_x/ypos values originate from estimated_source_in_slit_x/y and hence the meta data are actually still there, just under a different name?

Reply:
That is correct.